src/core/Akka.Docs.Tests/Configuration/ConfigurationSample.cs

```diff
@@ -207,7 +207,7 @@
 //        [Fact]
 //        public void UnsolvableSubstitutionWillThrowSample()
```

Comment:
Remove this test, its a duplicate of the one you've made below

Reply:
Done